packages/api/cms-api/src/index.ts

```diff
@@ -59,6 +61,7 @@ export { PaginatedResponseFactory } from "./common/pagination/paginated-response
 export { SortArgs } from "./common/sorting/sort.args";
 export { SortDirection } from "./common/sorting/sort-direction.enum";
 export { IsNullable } from "./common/validators/is-nullable";
+export { IsPhoneNumber, isPhoneNumber, IsPhoneNumberConstraint } from "./common/validators/is-phone-number";
```

Comment:
```suggestion
export { IsPhoneNumber, isPhoneNumber } from "./common/validators/is-phone-number";
```

I don't think it's necessary to export the constraint. Should we even export the validator? If yes, a changeset would be nice.